### PR TITLE
adding local Activity in php

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
Added local Activity in PHP to the app gen guide.

Then I ran prettier.

### CODE REVIEW

File: docs/application-development-guide.md [hunk 1]
```diff
@@ -3132,6 +3132,55 @@ const handle = await client.start(scheduledWorkflow, {
...
+
+#[ActivityInterface]
+interface YourActivityInterface
+{
+    #[ActivityMethod("youractivity")]
+    public function youractivity(): string;
+}
+```
+
+Local activity requires `LocalActivityOptions`:
```
Comment (DJSanti) [inline docs/application-development-guide.md:3165]: Be consistent with capitalizing 'Activity.'

File: docs/application-development-guide.md [hunk 2]
```diff
@@ -3132,6 +3132,55 @@ const handle = await client.start(scheduledWorkflow, {
...
+
+```php
+$greetingActivity = Workflow::newActivityStub(
+    YourActivityInterface::class,
+    LocalActivityOptions::new()->withStartToCloseTimeout(\DateInterval::createFromDateString('30 seconds'))
+);
+```
+
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`.
```
Comment (DJSanti) [inline docs/application-development-guide.md:3174]: When you have a list of three or more items, put it in a bulleted list.

File: docs/application-development-guide.md [hunk 3]
```diff
@@ -3132,6 +3132,55 @@ const handle = await client.start(scheduledWorkflow, {
...
+<TabItem value="typescript">
+
+Content is not available
+
+</TabItem>
+</Tabs>
+
 ## Observability
 
 TODO
```
Comment (DJSanti) [inline docs/application-development-guide.md:3186]: If this is going to stay in the release, change it to something like:

"This section is in progress."

Something more than just TODO

Comment (DJSanti) [reply docs/application-development-guide.md:3186]: This goes for all TODOs later on

File: docs/php/activities.md [hunk 4]
```diff
@@ -287,7 +287,7 @@ public function greet(string $name): \Generator
 ## Async completion
 
 There are certain scenarios when moving on from an Activity upon completion of its function is not possible or desirable.
-For example, you might have an application that requires user input in order to complete the Activity.
+For example, you might have an application that requires user input to complete the Activity.
 You could implement the Activity with a polling mechanism, but a simpler and less resource-intensive implementation is to asynchronously complete a Temporal Activity.
```
Comment (DJSanti) [inline docs/php/activities.md:291]: Move this up a space.

File: docs/php/activities.md [hunk 5]
```diff
@@ -314,7 +314,7 @@ $activityClient->completeExceptionallyByToken($taskToken, new \Error("activity f
 
 ## Local Activity
 
-In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local 
+In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local
```
Comment (DJSanti) [inline docs/php/activities.md:317]: Get rid of the first sentence, and hyperlink 'Local Activity' in the next sentence.

File: docs/php/activities.md [hunk 6]
```diff
@@ -337,4 +337,4 @@ $greetingActivity = Workflow::newActivityStub(
 );
 ```
 
-Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`. 
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`.
```
Comment (DJSanti) [inline docs/php/activities.md:340]: Same thing with the lists. Bulleted.

File: docs/php/how-to-create-local-activities-in-php.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+Local activity requires `LocalActivityOptions`:
+
+```php
+$greetingActivity = Workflow::newActivityStub(
+    YourActivityInterface::class,
+    LocalActivityOptions::new()->withStartToCloseTimeout(\DateInterval::createFromDateString('30 seconds'))
+);
+```
+
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`.
```
Comment (DJSanti) [inline docs/php/how-to-create-local-activities-in-php.md:29]: Up to you. You're probably sick of my opinions on lists of items.

**Review by DJSanti — COMMENTED** [review]
Definitely see if someone else can run through for conciseness. Other than that, it should do.

File: docs/application-development-guide.md (hunk 3 above)
Comment (flossypurse) [reply docs/application-development-guide.md:3186]: Ya, I agree with that.
I have a bad habit of leaving "todo" in places.
I will clean up my act on this too.

File: docs/php/how-to-create-local-activities-in-php.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-create-local-activities-in-php
```
Comment (flossypurse) [inline docs/php/how-to-create-local-activities-in-php.md:2]:
```suggestion
id: how-to-execute-a-local-activity-in-php
```

File: docs/php/how-to-create-local-activities-in-php.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+Local Activities requires `LocalActivityOptions`:
+
+```php
+$greetingActivity = Workflow::newActivityStub(
+    YourActivityInterface::class,
+    LocalActivityOptions::new()->withStartToCloseTimeout(\DateInterval::createFromDateString('30 seconds'))
+);
+```
+
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, and `RetryPolicy`.
```
Comment (flossypurse) [inline docs/php/how-to-create-local-activities-in-php.md:29]:
```suggestion
Local Activity Execution options include the following: 

- `ScheduleToCloseTimeout`
- `StartToCloseTimeout`
- `RetryPolicy`.
```

File: docs/php/activities.md (hunk 6 above)
Comment (flossypurse) [reply docs/php/activities.md:340]: Suggested change is below.

File: docs/php/how-to-create-local-activities-in-php.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-create-local-activities-in-php
+title: Create local Activities in PHP
+sidebar_label: Local Activity
+---
+
+To create a local Activity, use `#[LocalActivityInterface]`.
```
Comment (flossypurse) [inline docs/php/how-to-create-local-activities-in-php.md:7]:
```suggestion
To execute a Local Activity, use `#[LocalActivityInterface]`.
```

File: docs/php/how-to-create-local-activities-in-php.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-create-local-activities-in-php
+title: Create local Activities in PHP
```
Comment (flossypurse) [inline docs/php/how-to-create-local-activities-in-php.md:3]:
```suggestion
title: How to execute a Local Activity in PHP
```

File: guide-gen/guide-configs/app-dev.json [hunk 12]
```diff
@@ -1232,6 +1232,32 @@
...
+            },
+            {
+              "lang": "java",
+              "title": "Java",
+              "path": "none"
+            },
+            {
+              "lang": "php",
+              "title": "PHP",
+              "path": "php/how-to-create-local-activities-in-php"
```
Comment (flossypurse) [inline guide-gen/guide-configs/app-dev.json:1253]:
```suggestion
              "path": "php/how-to-execute-a-local-activity-in-php"
```

File: docs/application-development-guide.md (hunk 2 above)
Comment (flossypurse) [reply docs/application-development-guide.md:3174]: Suggested change below

**Review by flossypurse — APPROVED** [review]

**Pull request merged**